iterative/resource_task.go

```diff
@@ -396,3 +434,17 @@ func diagnostic(diags diag.Diagnostics, err error, severity diag.Severity) diag.
 		Summary:  err.Error(),
 	})
 }
+
+// parseContainerPath will attempt to separate the container name from the optional subdirectory
+// in the container.
+func parseContainerPath(raw string) (string /* container name */, string /* subdirectory */) {
+	parts := strings.SplitN(raw, "/", 2)
+	if len(parts) == 0 {
+		return "", ""
+	}
+	if len(parts) == 1 {
+		// No subdirectory specified.
+		return parts[0], ""
+	}
+	return parts[0], "/" + parts[1]
+}
```

Comment:
I'd rather avoid writing our own function for this and use the ones exposed by `rclone` public API as illustrated at https://github.com/iterative/terraform-provider-iterative/pull/687#discussion_r992891861.

Reply:
But we're not parsing an rclone string here, the referenced example returns an error when used with a simple `container/directory` string.

Reply:
I mean [`bucket.Split`](https://pkg.go.dev/github.com/rclone/rclone/lib/bucket#Split)

Reply:
Fixed in [b900f4b](https://github.com/iterative/terraform-provider-iterative/pull/687/commits/b900f4b9cd54d2a2cf899e0eacf1eeff560722ec)